CHANGELOG.md

```diff
@@ -1,3 +1,121 @@
+# v1.9.0 (2022-07-28)
+
+## OS Changes
+
+### General
+
+* SELinux policy now suppresses audit for tmpfs relabels ([#2222])
+* Restrict permissions for `/boot` and `System.map` ([#2223])
+* Remove unused crates `growpart` and `servicedog` ([#2238])
+* Migrate to using `aws-sdk-rust` for first-party OS rust packages ([#2300])
+* New mount in host containers for system logs ([#2295])
+* Add support for AWS China regions ( [#2224], [#2242], [#2247], [#2285])
```

Comment:
Maybe add an `### AWS` category, like we have for VMware, to collect the two AWS specific items.